.github/workflows/monorepo-docker.yml

```diff
@@ -8,6 +8,7 @@ on:
     paths:
       # For now, because this image is only used to use `infra`, we just build for infra changes
       - 'typescript/infra/**'
+      - 'typescript/sdk/**'
```

Comment:
Should all package be included here (utils, core)?

Reply:
added them, ty